Exercise_5.java

```diff
@@ -1,20 +1,82 @@
 class IterativeQuickSort { 
     void swap(int arr[], int i, int j) 
     { 
-	//Try swapping without extra variable 
+    	//Try swapping without extra variable 

```

Comment:
Good Job! Shardul. Please mention time and space complexities. Also add comments explaining your implementation.